cli/crates/server/src/servers.rs

```diff
@@ -184,16 +184,17 @@ async fn spawn_servers(
         ]
     }));
 
-    let miniflare = Command::new("node")
+    let mut miniflare = Command::new("node");
+    miniflare
         // Unbounded worker limit
         .env("MINIFLARE_SUBREQUEST_LIMIT", "1000")
         .args(miniflare_arguments.iter().map(std::convert::AsRef::as_ref))
         .stdout(if tracing { Stdio::inherit() } else { Stdio::piped() })
         .stderr(if tracing { Stdio::inherit() } else { Stdio::piped() })
         .current_dir(&environment.user_dot_grafbase_path)
-        .kill_on_drop(watch)
-        .spawn()
-        .map_err(ServerError::MiniflareCommandError)?;
+        .kill_on_drop(watch);
+    trace!("Spawning {miniflare:?}");
```

Comment:
What's this used for?

Reply:
A few times already miniflare failed to start on my machine, for one reason or another (usually when scripts are being refactored). This helps me see what command is actually executed. If you think this adds noise I will drop it.

Reply:
Could you screenshot the output please?

Reply:
Should be fine but would like to see how verbose it is

Reply:
```
2023-04-04T10:47:38.974477Z TRACE grafbase: subcommand: dev
Grafbase CLI 0.18.0-rc.4

2023-04-04T10:47:38.974620Z TRACE grafbase::dev: attempting to start server
2023-04-04T10:47:38.975703Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: validating dependencies
2023-04-04T10:47:38.975788Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: validating Node.js version
2023-04-04T10:47:38.975798Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: minimal supported Node.js version: v18.0.0
2023-04-04T10:47:38.979225Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: installed node version: v18.12.1
2023-04-04T10:47:38.979333Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: parsing schema
2023-04-04T10:47:38.979555Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: using a temporary file for the parser output: /tmp/.tmpfBN4HF
2023-04-04T10:47:39.046939Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: waiting for bridge ready
2023-04-04T10:47:39.046994Z TRACE grafbase_local_server::bridge::server: starting bridge at port 49155
2023-04-04T10:47:39.048600Z TRACE grafbase_local_server::bridge::listener::server: starting db event listener
2023-04-04T10:47:39.048716Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: bridge ready
2023-04-04T10:47:39.048734Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: spawning miniflare for the main worker
2023-04-04T10:47:39.048762Z TRACE spawn_servers{worker_port=4000 bridge_port=49155 watch=true sender=Sender { .. } event_bus=broadcast::Sender tracing=true}: grafbase_local_server::servers: Spawning Command { std: cd "/home/tomas/.grafbase" && MINIFLARE_SUBREQUEST_LIMIT="1000" "node" "--experimental-vm-modules" "./node_modules/miniflare/dist/src/cli.js" "--modules" "--host" "127.0.0.1" "--port" "4000" "--no-update-check" "--no-cf-fetch" "--do-persist" "--wrangler-config" "./wrangler.toml" "--binding" "BRIDGE_PORT=49155" "--text-blob" "REGISTRY=/home/tomas/work/workspaces/grafbase/grafbase/cli/.grafbase/registry.json" "--mount" "stream-router=./stream-router", kill_on_drop: true }
📡 Listening on port 4000

```

Perhaps more generally we could tweak the log levels so that we don't show all of this on `-t 1`.

Reply:
Seems fine